packages/button/constant.ts

```diff
@@ -0,0 +1,33 @@
+// The MIT License
+//
+// Copyright (c) 2019 Google, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+const CSS_CLASSES = {
+  ROOT: 'mdc-button',
+  ICON: 'mdc-button__icon',
+  LABEL: 'mdc-button__label',
+  DENSE: 'mdc-button--dense',
+  RAISED: 'mdc-button--raised',
+  OUTLINED: 'mdc-button--outlined',
+  UNELEVATED: 'mdc-button--unelevated',
+};
+
+export {CSS_CLASSES};
```

Comment:
`cssClasses`?

Reply:
I spoke to @williamernest about this. We decided that we should follow what is in the Google TypeScript style guide, which states enums should be all uppercase and `_` separated.